common/utils.go

```diff
@@ -42,6 +42,16 @@ func StringToHash(chainID int64, hash string) ([]byte, error) {
 	return nil, fmt.Errorf("cannot convert hash to bytes for chain %d", chainID)
 }
 
+// GasPriceMultiplier returns the gas price multiplier for the given chain
+func GasPriceMultiplier(chainID int64) float64 {
+	if IsEVMChain(chainID) {
+		return EVMOuttxGasPriceMultiplier
+	} else if IsBitcoinChain(chainID) {
+		return BTCOuttxGasPriceMultiplier
+	}
+	return 1.0
```

Comment:
Should we also return an error from this function? I don't think we would ever be using the 1.0 value , would we ?

Reply:
https://github.com/zeta-chain/node/pull/1881/commits/b396ccb2db2dfdf16cfa02092d66de12b2c15c20